config/routes.php

```diff
@@ -12,9 +12,11 @@
 use Yiisoft\Router\Route;
 use Yiisoft\Swagger\Middleware\SwaggerJson;
 use Yiisoft\Swagger\Middleware\SwaggerUi;
+use Yiisoft\Yii\Sentry\SentryMiddleware;
 
 return [
     Route::get('/')
+        ->middleware(SentryMiddleware::class)
```

Comment:
Shouldn't it be in the app middlware instead? https://github.com/yiisoft/app-api/blob/master/config/web/application.php#L21

Reply:
Of course. I just didn't find that place :)